lib.sh

```diff
@@ -445,6 +445,30 @@ dev_lettuce() {
     exit $rval
 }
 
+reexport_test_zips() {
+  ZIPFILES=rdrf/rdrf/features/exported_data/*.zip
+  for f in $ZIPFILES
+  do
+      f2="/app/$f"
+      reexport_test_zip $f2
+  done
+}
+
+reexport_test_zip() {
+    info 'reexport test zips'
+    _start_test_stack --force-recreate -d
+
+    set -x
+    set +e
+    docker-compose --project-name ${PROJECT_NAME} -f docker-compose-unittests.yml run testhost /app/docker-entrypoint.sh /app/reexport_zip.sh $1
```

Comment:
Should this be an exec rather than a run given there is a complete stack already running?
If it has to be a run, then perhaps add --rm